langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/QueryResponse.java

```diff
@@ -0,0 +1,34 @@
+package dev.langchain4j.store.embedding.chroma;
+
+import java.util.List;
+import java.util.Map;
+
+class QueryResponse {
+
+    private List<List<String>> ids;
```

Comment:
minor: please have a consistent ordering of parameters in all places (e.g. AddEmbeddingsRequest)

Reply:
done